flexible-classroom/get-started/enable-flexible-classroom.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Enable Flexible Classroom"
```

Comment:
Suggestion: Reorder the articles in the get-started menu. I think it's better to show "Enable Flexible Classroom", "Configure Flexible Classroom", and "Quickstart" in sequence, which is also the user journey. The user needs to enable and configure this product on Agora Console first and then run Android or whatever projects.
Reason: Adapt to the user journey.

Reply:
I put the configure into the enable section. One doc, done.